docs/README.md

```diff
@@ -1,154 +1,233 @@
-    # Then go through the Merge commit linked to PR's (making sure they apply
-    to that release)
-    # see https://github.com/docker/docker/commits/master
-    git cherry-pick -x fe845c4
-    # Repeat until you have cherry picked everything you will propose to be merged
-    git push upstream post-1.2.0-docs-update-1
+    	$ git push origin post-1.2.0-docs-update-1
 
-Then make a pull request to merge into the `docs` branch, __NOT__ into master.
+9. Make a pull request to merge into the `docs` branch.
 
-Once the PR has the needed `LGTM`s, merge it, then publish to our beta server
-to test:
+	Do __NOT__ merge into `master`.
+	
+10. Have mainters review your pull request.
 
-    git fetch upstream
-    git checkout docs
-    git reset --hard upstream/docs
-    make AWS_S3_BUCKET=beta-docs.docker.io BUILD_ROOT=yes docs-release
+11. Once the PR has the needed `LGTM`s, merge it on GitHub.
 
-Then go to http://beta-docs.docker.io.s3-website-us-west-2.amazonaws.com/
-to view your results and make sure what you published is what you wanted.
+12. Return to your local fork and make sure you are still on the `docs` branch.
 
-When you're happy with it, publish the docs to our live site:
+		$ git checkout docs
 
-    make AWS_S3_BUCKET=docs.docker.com BUILD_ROOT=yes DISTRIBUTION_ID=C2K6......FL2F docs-release
+13. Fetch the your merged pull request from `docs`.
 
-Test the uncached version of the live docs at http://docs.docker.com.s3-website-us-east-1.amazonaws.com/
+		$ git fetch upstream/docs
+	
+14. Ensure your branch is clean and set to the latest.
+
+   	 	$ git reset --hard upstream/docs
+    
+15. Copy the `awsconfig` file into the `docs` directory.
     
-Note that the new docs will not appear live on the site until the cache (a complex,
-distributed CDN system) is flushed. The `make docs-release` command will do this
-_if_ the `DISTRIBUTION_ID` is set to the Cloudfront distribution ID (ask the meta
-team) - this will take at least 15 minutes to run and you can check its progress
-with the CDN Cloudfront Chrome addin.
+16. Make the beta documentation
+
+    	$ make AWS_S3_BUCKET=beta-docs.docker.io BUILD_ROOT=yes docs-release
+
+17. Go to [http://beta-docs.docker.io.s3-website-us-west-2.amazonaws.com/] site
+to view your results.
+
+18. Make sure what you published is what you wanted.
+
+19. When you're happy with it, publish the docs to our live site:
+
+   		$ make AWS_S3_BUCKET=docs.docker.com BUILD_ROOT=yes
+DISTRIBUTION_ID=C2K6......FL2F docs-release
+
+20. Test the uncached version of the live docs at [http://docs.docker.com.s3-website-us-east-1.amazonaws.com/]
+
+
+### Caching and the docs
+
+New docs do not appear live on the site until the cache (a complex, distributed
+CDN system) is flushed. The `make docs-release` command flushes the cache _if_
+the `DISTRIBUTION_ID` is set to the Cloudfront distribution ID. The cache flush 
+can take at least 15 minutes to run and you can check its progress with the CDN
+Cloudfront Chrome addin.
```

Comment:
They refer to it as an extension, http://www.cdnplanet.com/tools/cloudfront-purge-tool/. Probably doesn't really matter though.